aptos-move/framework/aptos-names/sources/domains.move

```diff
@@ -8,6 +8,7 @@ module aptos_names::domains {
     use aptos_names::time_helper;
     use aptos_names::token_helper;
     use aptos_names::utf8_utils;
+    use aptos_names::verify;
     use aptos_std::event;
```

Comment:
This should be `aptos_framework::event`. Be sure to fix it everywhere else too!

Reply:
I see a lot of `aptos_std::event` in the codebase, e.g. [this one](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/coin.move#L11). what's the difference between `aptos_framework::event` and `aptos_std::event`?

Reply:
Hehe, the "difference" is someone forgot to rename `aptos_std::event` to `aptos_framework::event` when they moved `event.move` into `aptos_framework/`...

Fortunately, this did not break anything (except CLion's error handling), since both `aptos_std` and `aptos_framework` are aliases for address `0x01`

Reply:
ahh ok makes sense, thanks for explaining. will create a separate PR for this since this involves changing multiple other files (e.g.  `coin.move`, `reconfiguration.move`, etc.)

Reply:
see [this PR](https://github.com/aptos-labs/aptos-core/pull/4929) for changing all `aptos_std::events` to `aptos_framework::events`!